crates/gitbutler-tauri/src/stack.rs

```diff
@@ -134,5 +135,6 @@ pub fn push_stack_to_review(
     let ctx = CommandContext::open(&project, settings.get()?.clone())?;
     gitbutler_sync::stack_upload::push_stack_to_review(&ctx, &user, stack_id)?;
 
+    emit_vbranches(&windows, project_id);
```

Comment:
This is an unrelated but important change that propagates newly assigned review_ids to the frontend